test/e2e/network/dns_configmap.go

```diff
@@ -48,7 +48,8 @@ var _ = SIGDescribe("DNS configMap federations [Feature:Federation]", func() {
 
 	t := &dnsFederationsConfigMapTest{dnsTestCommon: newDNSTestCommon()}
 
-	ginkgo.It("should be able to change federation configuration [Slow][Serial]", func() {
+	// marked as disruptive because it can put coredns in a crashloop state
```

Comment:
Can we add some comments saying why the crashloop state can happen? Is it because the default CoreDNS configmap is overwritten for the duration of the test and may not be restored or something else?